src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const arrString = sourceString.split(';');
+
+  for (const item of arrString) {
+    if (item.trim() !== '') {
+      const [key, value] = item.split(':');
+
+      if (key && value) {
+        result[key.trim()] = value.trim();
+      }
+    }
+  }
```

Comment:
[CODE KNOWLEDGE] - The code is using a loop to iterate over the array `arrString`. According to the checklist, you should avoid using loops and use iteration methods instead. You could use `reduce` to transform the array of style declarations into an object.